### ISSUE TITLE
Airline theme support?

### ISSUE THREAD

**Issue by rlue** [karoliskoncevicius/sacredforest-vim#7]
Love the theme. Curious to know if the author is willing to accept a PR for Airline support.

**Comment by karoliskoncevicius** [comment]
Hello and thank you for good words about the theme.

Yup, I would be willing to accept airline support. I did the same for lightline here: https://github.com/karoliskoncevicius/sacredforest-vim/pull/3

However, please note that I have no experience with neither lightline nor airline, so will not be able to make any judgements or suggestions. Except that maybe it would be nice if airline and lightline styles were somewhat consistent?

**Comment by karoliskoncevicius** [comment]
Hello @rlue, I merged your pull request with a couple of changes, mainly made sure that all the used colors within airline are chosen from the set of available colors in this theme. This is to make sure that people who choose to alter their terminal emulator color codes get the exact same result as those who use "termguicolors" or "gvim".

To do this I had to lighten the background are in the middle of status line. Curious to hear your opinion if this is acceptable.
